src/trousse/row_fix.py

```diff
@@ -310,9 +323,76 @@ def print_errors_per_column(self):
         print("The errors per feature are:")
         for c in self.errors_before_correction_dict.keys():
             print(
-                f"{c}: {len(self.errors_before_correction_dict[c])} : {set(self.errors_before_correction_dict[c])}"
-                f" ---> {len(self.errors_after_correction_dict[c])} : {set(self.errors_after_correction_dict[c])}"
+                f"{c}: {len(self.errors_before_correction_dict[c])} :"
+                f" {set(self.errors_before_correction_dict[c])}"
+                f" ---> {len(self.errors_after_correction_dict[c])} :"
+                f" {set(self.errors_after_correction_dict[c])}"
+            )
+
+    @staticmethod
+    def forced_conversion_to_numeric(
+        df_info: DataFrameWithInfo,
+        columns: Iterable,
+        dry_run: bool = False,
+        verbose: bool = True,
+    ) -> DataFrameWithInfo:
+        """
+        Convert ``columns`` to a numeric type. When not possible set values to NaN.
+
+        This method also logs the number of values that are not number-convertible,
+        before setting them to NaN.
+
+        Parameters
+        ----------
+        df_info : DataFrameWithInfo
+            Instance with the data that will be analyzed and set to NaN
+        columns : Iterable
+            List of columns of ``df_info`` that will be analyzed and modified.
+            Usually it can be set to "mixed-type" columns.
```

Comment:
```suggestion
            Usually it can be set to mixed-type columns.
```
I'm removing the quotes because it seems like it is the name of a column or something like this